langchain4j-mcp/src/main/java/dev/langchain4j/mcp/client/DefaultMcpClient.java

```diff
@@ -71,7 +72,8 @@ public class DefaultMcpClient implements McpClient {
 
     public DefaultMcpClient(Builder builder) {
         transport = ensureNotNull(builder.transport, "transport");
-        clientName = getOrDefault(builder.clientName, () -> "langchain4j-" + UUID.randomUUID());
+        key = getOrDefault(builder.key, () -> "langchain4j-" + UUID.randomUUID());
```

Comment:
```suggestion
        key = getOrDefault(builder.key, () -> UUID.randomUUID());
```
this prefix seems to be unnecessary?

Reply:
Removed.